io/www/src/fragment/odinSchemaTransform.js

```diff
@@ -0,0 +1,143 @@
+//utitilies to transform payload from old schema to new schema
+
+const CF_REFERENCE_FIELDS = ['cards', 'collections', 'entries'];
+const REFERENCE_FIELDS = [...CF_REFERENCE_FIELDS, 'tags'];
+
+function transformFields(body) {
+    const { fields, references } = body;
+    const pathToIdMap = {};
+    if (references) {
+        references.forEach((reference) => {
+            const { type, path, id } = reference;
+            if (type === 'content-fragment') {
+                pathToIdMap[path] = id;
+            }
+        });
+    }
+    const transformedFields = fields.reduce(
+        (fields, { name, multiple, values, mimeType }) => {
```

Comment:
was comparing it with https://github.com/adobecom/milo/blob/stage/libs/features/mas/src/aem-fragment.js#L256 and you might miss settings?